frontend/views/profile.html

```diff
@@ -0,0 +1,95 @@
+<div class="me" ng-hide="!user">
+  <div ng-hide="edit" class="pull-right btn-group" role="group">
+    <button class="btn btn-default" type="submit" ng-click="editProfile()">Update</button>
+    <button class="btn btn-warning" type="submit" ng-click="logout()">Logout</button>
+  </div>
+  <div class="page-header">
+    <h2>
+      {{user.username}}
+    </h2>
+  </div>
+  <div ng-hide="edit" class="row">
+    <div class="col-sm-10">
+      <dl class="dl-horizontal">
+        <dt>Username</dt><dd>{{user.username}}</dd>
+        <dt>Email</dt><dd>{{user.email}}</dd>
+
+        <dt ng-hide="!user.catchphrase">Catchphrase</dt>
```

Comment:
I thought it made it clearer, but I'll switch it if that doesn't match your convention.